### PR TITLE
UI Fixes and Tests

### PR DESCRIPTION
**Pull request by roblim**
- https://app.asana.com/0/1119694864223484/1132621358605327/f
- https://app.asana.com/0/1119694864223484/1134557806509893/f
- https://app.asana.com/0/1119694864223484/1134557806509903/f

### CODE REVIEW

File: docs/guides/data_documentation.rst [hunk 1]
```diff
@@ -182,3 +182,13 @@ for how to profile a single batch of data and build documentation from the valid
   safe_mmkdir(os.path.dirname(profiling_html_filepath))
   with open(profiling_html_filepath, 'w') as writer:
       writer.write(DefaultJinjaPageView.render(document_model))
+
+
+Dependencies
```
Comment (abegong) [inline docs/guides/data_documentation.rst:187]: :+1:

File: great_expectations/render/view/templates/page.j2 [hunk 2]
```diff
@@ -160,7 +160,6 @@
       </div>
     </div>
     <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.15.0/umd/popper.min.js"></script>
```
Comment (abegong) [inline great_expectations/render/view/templates/page.j2:163]: Are you sure we don't need popper? I thought it was required for tooltips. Does Taylor's `cooltip` avoid this dependency?

File: tests/render/test_column_section_renderer.py [hunk 3]
```diff
@@ -427,114 +520,3 @@ def test_ValidationResultsTableContentBlockRenderer_generate_expectation_row_hap
         ]
     }
 
-def test_ValidationResultsTableContentBlockRenderer_generate_expectation_row_with_errored_expectation():
```
Comment (abegong) [inline tests/render/test_column_section_renderer.py:430]: Why remove this test?

Comment (abegong) [reply tests/render/test_column_section_renderer.py:430]: I see it. nvm

**Review by abegong — DISMISSED** [review]
This looks great. I have one question about popper, but it doesn't need to block.

File: tests/render/test_render_ValidationResultsTableContentBlockRenderer.py [hunk 4]
```diff
@@ -0,0 +1,1106 @@
+
+import json
```
Comment (jcampbell) [inline tests/render/test_render_ValidationResultsTableContentBlockRenderer.py:2]: I really like moving toward more robust tests here! A few small changes:

1. I think evr_succes, evr_failed, and evr_exception should refactored into fixtures since they appear to be reused.
2. evr_exception has a real user path in its traceback, which should be anonymized.
3. I found the long json blocks hard to follow (as in, I'm sure I didn't follow them). Could you add a small one-line comment around the intent of the specific test when there are several tests in a row (and if it's not a pain in your IDE fold up the json unless its important to read it specifically).

**Review by jcampbell — CHANGES_REQUESTED** [review]
One small set of requests on test inline!

File: great_expectations/render/view/templates/page.j2 (hunk 2 above)
Comment (roblim) [reply great_expectations/render/view/templates/page.j2:163]: Yep. It was required for tooltips when we were using bootstrap tooltips, but Taylor's cooltip implementation is pure CSS.

**Review by jcampbell — APPROVED** [review]
